sbin/build.sh

```diff
@@ -459,6 +463,41 @@ executeTemplatedFile() {
     exit 2
   fi
 
+  # Restore exit behavior
+  set -eu
+}
+
+createOpenJDKFailureLogsArchive() {
+    echo "OpenJDK make failed, archiving make failed logs"
+    cd build/*
```

Comment:
Are we sure we're in the directory that build hangs off?

Reply:
Yes called from executeTemplatedFile() which steps into working dir